dotnet/samples/GettingStartedWithAgents/Step08_Declarative.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ClientModel;
+using Azure.AI.Projects;
+using Azure.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.Agents.AzureAI;
+using Microsoft.SemanticKernel.Agents.OpenAI;
+using Microsoft.SemanticKernel.ChatCompletion;
+using OpenAI;
+
+namespace GettingStarted;
+
+/// <summary>
+/// This example demonstrates how to declaratively create instances of <see cref="KernelAgent"/>.
+/// </summary>
+public class Step08_Declarative : BaseAgentsTest
+{
+    public Step08_Declarative(ITestOutputHelper output) : base(output)
+    {
+        var openaiClient =
+           this.UseOpenAIConfig ?
+               OpenAIAssistantAgent.CreateOpenAIClient(new ApiKeyCredential(this.ApiKey ?? throw new ConfigurationNotFoundException("OpenAI:ApiKey"))) :
+               !string.IsNullOrWhiteSpace(this.ApiKey) ?
+                   OpenAIAssistantAgent.CreateAzureOpenAIClient(new ApiKeyCredential(this.ApiKey), new Uri(this.Endpoint!)) :
+                   OpenAIAssistantAgent.CreateAzureOpenAIClient(new AzureCliCredential(), new Uri(this.Endpoint!));
+
+        var aiProjectClient = AzureAIAgent.CreateAzureAIClient(TestConfiguration.AzureAI.ConnectionString, new AzureCliCredential());
```

Comment:
This sounds strongly dependant on Azure, should be changed or moved to AzureAIAgent ?

```suggestion
        var aiProjectClient = AzureAIAgent.CreateAzureAIClient(TestConfiguration.AzureAI.ConnectionString, new AzureCliCredential());
```

Reply:
The sample shows how to use the aggregator to handle any type of agent hence the need for the  AzureAI specific code. I can move it to a seperate MultiAgent sample.